src/style.css

```diff
@@ -0,0 +1,134 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --primary-color: #060b35;
+  --secondary-color: #616070;
+  --button-color: #00acdc;
+  --light-color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+h3,
+h4 {
+  margin: 0;
```

Comment:
it will be better to create a separate .css file and put all resets to it. After that, just import it in our main style.css.